typings/@chainlink/index.d.ts

```diff
@@ -105,4 +105,5 @@ declare module '@chainlink/types' {
 }
 declare module '@chainlink/ea-bootstrap'
 declare module '@chainlink/external-adapter'
+declare module 'synthetix'
```

Comment:
Don't know if this is correct

Reply:
Yes, this is ok. We need to declare it as it's a JS package, not TS.